### DISCUSSION TITLE
edit form from repeater

### DISCUSSION THREAD

**Discussion by mondherinfo** [filamentphp/filament#6188] [General]
I extend a new field from the repeater to edit its view and I override the delete item function, I want to recalculate the totalamount when I delete an item from the repeater, 

![image](https://user-images.githubusercontent.com/15198800/231603023-020c36c7-c163-4841-9cc6-c6c829fffb2c.png)

**Comment by danharrin** [comment]
Don't calculate it like that, calculate it dynamically with a Placeholder - https://stackoverflow.com/a/70836831
